### ISSUE TITLE
Kerasmodel keraszeropadding error

### ISSUE THREAD

**Issue by stig11** [deeplearning4j/deeplearning4j#3978]
#### Issue Description

Hi, I am trying to import a keras model into dl4j and get the below error:

```
Exception in thread "main" java.lang.ClassCastException: java.util.ArrayList cannot be cast to java.lang.Integer
	at org.deeplearning4j.nn.modelimport.keras.layers.KerasZeroPadding.getPaddingFromConfig(KerasZeroPadding.java:114)
	at org.deeplearning4j.nn.modelimport.keras.layers.KerasZeroPadding.<init>(KerasZeroPadding.java:51)
	at org.deeplearning4j.nn.modelimport.keras.KerasLayer.getKerasLayerFromConfig(KerasLayer.java:247)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.helperPrepareLayers(KerasModel.java:190)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:164)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:97)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel$ModelBuilder.buildModel(KerasModel.java:576)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:195)
	at org.deeplearning4j.examples.dataexamples.ImportKerasConfig.main(ImportKerasConfig.java:23)

```

I am using a transfer learned resnet50 model for binary classification from keras.applications and when importing get this error.  I saw examples of zeropadding2d but couldn't find any examples of these specific errors.

Apologies if this is an easy question, I am new to dl4j and just wanted to make a quickproof of concept of a model working in dl4j.  I have also attached the summary of my model.

[modelSummary.txt](https://github.com/deeplearning4j/deeplearning4j/files/1263491/modelSummary.txt)

**Comment by agibsonccc** [comment]
Keras version and dl4j version?

**Comment by stig11** [comment]
Keras 2.0.5
dl4j 0.8

**Comment by maxpumperla** [comment]
@stig11 I'm here to your rescue: the quick fix is to specify a _pair_ for your `padding` arg in `ZeroPadding2D`. You only seem to provide a single integer. It's a pity that keras internally doesn't make this a pair before serialising, but what can you do?!

**Comment by maxpumperla** [comment]
@stig11 also note that with #3980, if you would like to work on master (or starting with tomorrow's snapshot) this is now solved.

**Comment by stig11** [comment]
Awesome thanks @maxpumperla

**Comment by stig11** [comment]
Hi again, I have tried pulling and re building today and am now using 0.9.1 however I am still getting the same error.  Is this the correct version or do I need to build from source

**Comment by agibsonccc** [comment]
@stig11 wrong version. You want *snapshots* which is 0.9.2-SNAPSHOT documented here: https://deeplearning4j.org/snapshots

**Comment by stig11** [comment]
Ahh, cool thanks
